rfcs/20180907-contrib-sunset.md

```diff
@@ -0,0 +1,282 @@
+| grid_rnn           |(qlzh727 ebrevdo) | delete                               |
+| hadoop             | yongtang (mrry)  | move to tensorflow/io?               |
+| hooks              | ispirmustafa     | delete (already in core)             |
+| hvx                | satok16          | delete (redundant with NNAPI)        |
+| igfs (#22194)      | dmitrievanthony  | move to tensorflow/io                |
+| ignite (#22210)    | dmitrievanthony  | move to tensorflow/io                |
+| image              |                  | partial move to tensorflow/contrib?  |
+| input_pipeline     | rohan100jain     | delete                               |
+| integrate          | shoyer *mcoram   | move to tensorflow/scientific?       |
+| kafka              | yongtang (mrry)  | move to tensorflow/io?               |
+| keras              | fchollet         | delete (moved to tf.keras)           |
+| kernel_methods    | petrosmol rostami | move to tensorflow/estimator?        |
+| kfac               | duckworthd       | delete (moved to tensorflow/kfac)    |
+| kinesis            | yongtang (mrry)  | move to tensorflow/io?               |
+| labeled_tensor     | shoyer           | delete                               |
+| layers             |                  | partial move to tensorflow/contrib?  |
+| learn              | wicke       | delete (replaced by tensorflow/estimator) |
+| legacy_seq2seq     | ebrevdo (qlzh727) | delete (replaced by seq2seq)        |
+| libsvm             |                  | delete (no owner)                    |
+| linalg             | rmlarsen langmore | delete (moved to core)              |
+| linear_optimizer   | petrosmol (karmel) | move to tensorflow/estimator       |
+| lite             | aselle  petewarden | move to core                         |
+| lookup          | ysuematsu (ebrevdo) | move to core                         |
+| losses             |                  | partial move to tensorflow/contrib   |
+| makefile           |  petewarden      | delete (RPI build now uses bazel)    |
+| memory_stats       | wujingyue        | delete                               |
+| meta_graph_transform |  petewarden    | delete                               |
+| metrics            | brainnoise       | delete (replaced with OO metrics)    |
+| mixed_precision    | protoget reedwm  | delete                               |
+| model_pruning      | suyoggupta       | move to core                         |
+| mpi                | (poxvoculi)      |                                      |
+| mpi_collectives    | *jthestness (poxvoculi) |                               |
+| nccl               | (tobyboyd)       | move essential parts to core         |
+| nearest_neighbor   |                  | delete                               |
+| nn                 |                  | partial move to tensorflow/contrib?  |
+| opt                | *joshburkart apassos | move to tensorflow/contrib?      |
```

Comment:
@martinwicke Hi, I'm interested in `opt` module. My colleague and I contributed codes for adamax optimizer, elastic average optimizer, and agn_optimizer. I would love to help if needed.